subprojects/common/graph/src/main/kotlin/com/avito/graph/SimpleOperation.kt

```diff
@@ -0,0 +1,25 @@
+package com.avito.graph
+
+open class SimpleOperation(
+    override val id: String,
+    override val duration: Double = 1.0,
+    override val predecessors: Set<String> = emptySet()
+) : Operation {
+
+    override fun toString(): String {
```

Comment:
Why do not you simply use `data class`?

Reply:
Sure, will do

Reply:
Fixed in https://github.com/avito-tech/avito-android/pull/985/commits/518d8ab1ef7375db97009368f70eac5c29fffa70